inventory_management_system_api/routers/v1/item.py

```diff
@@ -26,7 +26,10 @@
     response_description="The created item",
     status_code=status.HTTP_201_CREATED,
 )
-def create_item(item: ItemPostRequestSchema, item_service: ItemService = Depends()) -> ItemSchema:
+def create_item(
+    item_service: Annotated[ItemService, Depends(ItemService)],
+    item: ItemPostRequestSchema,
```

Comment:
```suggestion
    item: ItemPostRequestSchema,
    item_service: Annotated[ItemService, Depends(ItemService)],
```
This one was fine this way round, I think its best to have any body parameters first, then services, then any optional query parameters.